client/signup/main.jsx

```diff
@@ -571,6 +584,18 @@ class Signup extends Component {
 		return window.location.origin + path;
 	};
 
+	getFilteredQueryArgs = () => {
```

Comment:
Nit: A more meaningful name can be helpful here.
```suggestion
	getCurrentFlowSupportedQueryParams = () => {
```

Reply:
Woot! That's indeed a better naming. ~Stole~ Updated in https://github.com/Automattic/wp-calypso/pull/70371/commits/d4a4849b79a5fb5616e63e328a476e3af6f3e19e.